client/components/post-schedule/clock.jsx

```diff
@@ -38,17 +38,24 @@ function checkTimeValue( value ) {
 }
 
 class PostScheduleClock extends Component {
+	constructor( props ) {
+		super( props );
+
+		// bounds
+		this.setTime = this.setTime.bind( this );
```

Comment:
I prefer this since we can usually avoid the need for creating a custom constructor.